contracts/integrations/test/fuzz_tests/stake_management_test.ts

```diff
@@ -20,14 +21,15 @@ export class StakeManagementSimulation extends Simulation {
 
         const poolManagement = new PoolManagementSimulation(this.environment);
 
-        const actions = [
-            ...stakers.map(staker => staker.simulationActions.validStake),
-            ...stakers.map(staker => staker.simulationActions.validUnstake),
-            ...stakers.map(staker => staker.simulationActions.validMoveStake),
-            poolManagement.generator,
-        ];
+        const [actions, weights] = _.unzip([
+            ...stakers.map(staker => [staker.simulationActions.validStake, 0.3 / stakers.length]),
+            ...stakers.map(staker => [staker.simulationActions.validUnstake, 0.2 / stakers.length]),
+            ...stakers.map(staker => [staker.simulationActions.validMoveStake, 0.3 / stakers.length]),
+            [poolManagement.generator, 0.2],
+        ]) as [Array<AsyncIterableIterator<AssertionResult | void>>, number[]];
```

Comment:
anyone have a cleaner way to do this? 🤔

Reply:
@dorothy-zbornak 😬

Reply:
I honestly don't think it's that bad. Maybe just comment what's going on?